src/jimple-frontend/jimple-language.cpp

```diff
@@ -0,0 +1,192 @@
+#include <assert.h>
+#include <jimple-frontend/jimple-language.h>
+#include <util/message/format.h>
+#include <c2goto/cprover_library.h>
+languaget *new_jimple_language(const messaget &msg)
+{
+  return new jimple_languaget(msg);
+}
+bool jimple_languaget::final(contextt &context, const messaget &msg)
+{
+  msg.status("Adding cprover library");
+  add_cprover_library(context, msg);
+
+  add_intrinsics(context);
+  msg.status("Adding __ESBMC_main");
+  setup_main(context);
+  return false;
+}
+bool jimple_languaget::from_type(
+  const typet &,
+  std::string &,
+  const namespacet &)
+{
+  // TODO
```

Comment:
If this function is just a place holder and not meant to be called, `assert(!"TODO");` probably facilitates debugging in the future.